docs/src/cookbooks/cookbook.md

```diff
@@ -897,6 +897,31 @@ compile(new TooWideOrNarrowUInt(8, 2))
 compile(new TooWideOrNarrowUInt(8, 4))
 ```
 
+## How can I use Verilog "case equality" operators in Chisel?
+
+Verilog has "case equality" (`===`) and inequality (`!==`) operators.
+They are typically used to ignore unknown (`X`) values in assertions.
+
+Chisel does not support SystemVerilog `X` directly, but it is possible to check if a value is `X` with `chisel3.util.circt.isX`.
```

Comment:
Mega-nit, but `X` is referred to as a Verilog feature above, so this should probably be consistent with that.

```suggestion
Chisel does not support Verilog `X` directly, but it is possible to check if a value is `X` with `chisel3.util.circt.isX`.
```

Reply:
Agreed, thanks!